### DISCUSSION TITLE
Database Reader not doing anything

### DISCUSSION THREAD

**Discussion by EthanKC** [nextgenhealthcare/connect#5213] [Q&A] [answered]
I set up a database reader to pull some information from a postgresql table. The data is there, and I know that the connector can access the database, because I was able to pull the data in to create the SQL query. But when I deploy the connector, nothing happens. It's like there's nothing on the table, but I know there is. Any help would be appreciated. I've attached the channel (as a .txt cause it wouldn't accept .xml?), along with the info on the database table. 
[base64processing.csv](https://github.com/nextgenhealthcare/connect/files/8766765/base64processing.csv)
[Database Reader - HL7.txt](https://github.com/nextgenhealthcare/connect/files/8766771/Database.Reader.-.HL7.txt)

**Comment by EthanKC** [comment, marked as answer]
Never mind, I found an answer.

**Reply by pacmano1** [reply]
Great.  Share it.

**Reply by EthanKC** [reply]
Sorry, I didn't post it cause I didn't come up with it. I found the answer here: 
https://github.com/nextgenhealthcare/connect/issues/4602

Specifically this: 
<img width="708" alt="image" src="https://user-images.githubusercontent.com/100318963/170270597-e07e8fc9-7b1c-4719-898c-9fd0d33fd6bb.png">

I added 

> --add-modules=java.sql.rowset
--add-exports=java.base/com.sun.crypto.provider=ALL-UNNAMED
--add-exports=java.base/sun.security.provider=ALL-UNNAMED
--add-opens=java.base/java.lang=ALL-UNNAMED
--add-opens=java.base/java.lang.reflect=ALL-UNNAMED
--add-opens=java.base/java.math=ALL-UNNAMED
--add-opens=java.base/java.net=ALL-UNNAMED
--add-opens=java.base/java.security=ALL-UNNAMED
--add-opens=java.base/java.security.cert=ALL-UNNAMED
--add-opens=java.base/java.text=ALL-UNNAMED
--add-opens=java.base/java.util=ALL-UNNAMED
--add-opens=java.base/sun.security.pkcs=ALL-UNNAMED
--add-opens=java.base/sun.security.rsa=ALL-UNNAMED
--add-opens=java.base/sun.security.x509=ALL-UNNAMED
--add-opens=java.desktop/java.awt=ALL-UNNAMED
--add-opens=java.desktop/java.awt.color=ALL-UNNAMED
--add-opens=java.desktop/java.awt.font=ALL-UNNAMED
--add-opens=java.sql.rowset/com.sun.rowset=ALL-UNNAMED
--add-opens=java.xml/com.sun.org.apache.xalan.internal.xsltc.trax=ALL-UNNAMED

to the mcservice.vmoptions and mcserver.vmoptions files, and it started working.
